devel/pyright/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           node 1.0
+
+name                pyright
```

Comment:
@reneeotten Any concerns relative to the port name? While it matches the upstream project, the `py` prefix might make folks think it's a Python-based port.

I'm almost wondering if it should be named `node-pyright`?

But if you're fine with the name as-is, then I'm good too!  ;-)

Reply:
My 2¢: the fact that it’s written in Node is an implementation detail and not something users should know or care about, so I’d vote for just keeping “pyright”. Besides, if MS decides to one day RIIR we wouldn’t need to rename it :)

Reply:
> My 2¢: the fact that it’s written in Node is an implementation detail and not something users should know or care about, so I’d vote for just keeping “pyright”. Besides, if MS decides to one day RIIR we wouldn’t need to rename it :)

Yep, and so long as others are fine with it, I'm good too. But just wanted to bring it up, to see if anyone feels differently.

Reply:
I think we should follow the upstream name; Python ports are name `py-<name>` so I don't think there is any confusion here.

Reply:
> I think we should follow the upstream name; Python ports are name `py-<name>` so I don't think there is any confusion here.

Sounds good to me. Thanks for the feedback Renee!